### PR TITLE
Create Web-One sidebar component

### PR DESCRIPTION
**Pull request by ethanleiter**
Creates new component "Web-One Sidebar" that enables user to add a sidebar to a page and input child elements within the sidebar.

### CODE REVIEW

File: dev/public/pages/webSidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    import WebSidebar from 'csun-react-components';
+    
+    <WebSidebar>
+    <h3> Test Header </h3>
+    <h4> Test Title </h4>
+    <p> Example paragraph for WebSidebar component. </p>
+  </WebSidebar>
+    
+  );
+
```
Comment (kevmojay) [inline dev/public/pages/webSidebar.jsx]: Spacing is off and line 14 shouldn't be there

File: src/components/web/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './sass/_webSidebar.scss';
+
+const WebSidebar = ({ children }) => <div className="sidebar-container">{children} </div>;
```
Comment (kevmojay) [inline src/components/web/Sidebar.jsx]: Experiment with using bootstrap grid system. (Same thing as MetaColumn)

Currently your setup doesn't support content to the side of it. Make sure you component works as expected with other content. Notice the "Sample Text" in the screenshot"

screenshot-2017-9-26 react app

**Comment by ethanleiter** [comment]
Updated so WebSidebar has matching height columns using Flexbox.

File: dev/public/pages/webSidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  const propTypesTitle = '// Prop Types';
+  const propTypes = `
+    children: PropTypes.node.isRequired,
+ `;
+
+  return (
+    <MetaContainer>
+      <MetaRow className={'flex'}>
+        <MetaColumn className={'flex'} size={['lg']} value={['3']}>
```
Comment (HeyItsMario) [inline dev/public/pages/webSidebar.jsx]: This line, line 29, 32... size and value props have become plural. So **sizes** and **values**

**Pull request merged**